fiftyone/operators/cache/manager.py

```diff
@@ -0,0 +1,36 @@
+"""
+Execution cache manager.
+
+| Copyright 2017-2025, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
+"""
+from fiftyone.operators.store import ExecutionStoreService
+from .utils import resolve_store_name, get_store_for_func
+
+
+def clear_all_caches():
+    """
+    Clears all named execution stores and their cache entries.
+    """
+    svc = ExecutionStoreService()
+    svc.clear_cache()
+
+
+def clear_function_cache(ctx, func, store_name=None, link_to_dataset=True):
```

Comment:
This might not make sense as a separate utility. We could keep all utils on the func itself, since a reference is required one way or another.